src/components/v2/transactionInfoModal.tsx

```diff
@@ -0,0 +1,332 @@
+  ETH: 'https://etherscan.io/tx/',
+  POLYGON: 'https://polygonscan.com/tx/',
+  BSC: 'https://bscscan.com/tx/',
+  AVALANCHE: 'https://explorer.avax.network/tx/',
+  ARBITRUM: 'https://arbiscan.io/tx/',
+  OPTIMISM: 'https://optimistic.etherscan.io/tx/',
+  FANTOM: 'https://ftmscan.com/tx/',
+  EVMOS: 'https://escan.live/tx/',
+  COSMOS: 'https://www.mintscan.io/cosmos/txs/',
+  OSMOSIS: 'https://www.mintscan.io/osmosis/txs/',
+  JUNO: 'https://www.mintscan.io/juno/txs/',
+  STARGAZE: 'https://www.mintscan.io/stargaze/txs/',
+  NOBLE: 'https://www.mintscan.io/noble/txs/'
+};
+
+const getTransactionDestinationDetails = (type: string, from: string, to: string) => {
+  let destination = 'Application';
+  let transactionAddress;
+  switch (type) {
+    case TransactionType.SELF:
+    case TransactionType.SEND:
+      destination = 'To';
+      transactionAddress = getMaskedAddress(to);
+      break;
+    case TransactionType.RECEIVE:
+      destination = 'From';
+      transactionAddress = getMaskedAddress(from);
+      break;
+    default:
+      transactionAddress = to;
+      if (APPLICATION_ADDRESS_NAME_MAP.has(transactionAddress)) {
+        transactionAddress = APPLICATION_ADDRESS_NAME_MAP.get(transactionAddress) as string;
+      } else {
+        transactionAddress = getMaskedAddress(transactionAddress);
+      }
+  }
+
+  return [destination, transactionAddress];
+};
+
+export default function TransactionInfoModal ({
+  isModalVisible,
+  setModalVisible,
+  params,
+  navigationRef
+}: {
+  isModalVisible: boolean
+  setModalVisible: React.Dispatch<React.SetStateAction<boolean>>
+  params: {
+    timestamp: string
+    blockchain: string
+    hash: string
+    gas: string
+    type: string
+    from: string
+    to: string
+    value: string
+    token: string | null
+    tokenIcon: string | null
+    fromToken: string | null
+    fromTokenValue: string | null
+    toToken: string | null
+    fromTokenIcon: string | null
+    toTokenIcon: string | null
+    status: string
+  } | null
+  navigationRef: any
+}) {
+  const viewRef = useRef();
+  const { t } = useTranslation();
+
+  const copyHash = (url: string) => {
+    copyToClipboard(url);
+    Toast.show(t('COPIED_TO_CLIPBOARD'));
+  };
+
+  if (params !== null) {
+    const { timestamp, blockchain, hash, gas, type, from, to, value, token, tokenIcon, fromToken, fromTokenValue, toToken, fromTokenIcon, toTokenIcon, status } = params;
+    const [destination, transactionAddress] = getTransactionDestinationDetails(type, from, to);
+    const chain = blockchain.charAt(0).toUpperCase() + blockchain.slice(1).toLowerCase();
+
+    const RenderTransactionHeader = () => {
+      const title = type ? type.charAt(0).toUpperCase() + type.slice(1).toLowerCase() : 'Unknown';
+      const displayTitle = title || 'Unknown Transaction';
+      return (
+        <CyDView>
+          <CyDText className='text-center font-nunito text-[20px] font-extrabold text-activityFontColor'>
```

Comment:
don't have to mention font-nunito when using CyDText.

Reply:
done ~ removed